omero/sysadmins/unix/server-installation.txt

```diff
@@ -270,19 +269,8 @@ If possible, install one of the following packages:
 
 The Ice version may vary, depending upon the distribution version you
 are using. The Ice versions in currently supported versions of Debian
-and `Ubuntu <https://wiki.ubuntu.com/Releases>`_ are shown below:
-
-+--------+--------------+-------------------+---------------+
-| Distribution          | ZeroC Ice version | OMERO version |
-+========+==============+===================+===============+
-| Debian | 7.0          | 3.4               | 4.4.x, 5.x.x  |
-+--------+--------------+-------------------+---------------+
-| Ubuntu | 14.04 (LTS)  | 3.5               | 4.4.x, 5.x.x  |
-|        +--------------+-------------------+---------------+
-|        | 13.10        | 3.4               | 4.4.x, 5.x.x  |
-|        +--------------+-------------------+---------------+
-|        | 12.04 (LTS)  | 3.4               | 4.4.x, 5.x.x  |
-+--------+--------------+-------------------+---------------+
+and `Ubuntu <https://wiki.ubuntu.com/Releases>`_ are shown in the
+:ref:`ice-section` of the :doc:`/sysadmins/version-requirements` page.
```

Comment:
Maybe use `ice-requirements` or `ice-version-requirements` as the cross ref indentifer; no need to use `-section` since the cross-refs are to sections so that's already implied.